src/casus/blocks/AndBlock.js

```diff
@@ -1,13 +1,25 @@
 //@flow strict
 
 import BinaryOperationBlock from './BinaryOperationBlock.js';
+import BooleanValue from '../interpriter/BooleanValue.js';
```

Comment:
Interpreter is spelled incorrect I believe. Not a necessary fix, I'll leave it up to you.